MAUI/ListView/working-with-sflistview.md

```diff
@@ -0,0 +1,769 @@
+---
+layout: post
+title: Working with ListView in .NET MAUI ListView control | Syncfusion
+description: Learn here all about Working with ListView support in Syncfusion .NET MAUI ListView control and more.
+platform: MAUI
+control: SfListView
+documentation: ug
+---
+
+# Working with ListView in .NET MAUI ListView (SfListView)
+
+## Interacting with ListView Items
+
+### Loaded event
+
+The [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event is raised when the `SfListView` is loading in view for the first time.

```

Comment:
The [Loaded](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.ListView.SfListView.html#Syncfusion_Maui_ListView_SfListView_Loaded) event is triggered when the `SfListView` is loaded in view for the first time.

Reply:
I have modified the line as per the concerns provided